app/services/data_migrations/backfill_none_hesa_disabilities_codes.rb

```diff
@@ -0,0 +1,18 @@
+module DataMigrations
+  class BackfillNoneHesaDisabilitiesCodes
+    TIMESTAMP = 20210610111514
+    MANUAL_RUN = false
+
+    def change
+      no_disabilities_hesa_code = Hesa::Disability.find('no').hesa_code
+
+      ApplicationForm
```

Comment:
Do we want to avoid the  touching the records, also not having audits?

Reply:
had a think around this, and actually its best if we do, as we are updating the HESA code so good for integrations to pull the changes in. I'll check how many records this affects though